src/content/structured/get-started/react.mdx

```diff
@@ -103,3 +103,43 @@ module.exports = withTM(nextConfig);
 The `@ukic/react` package will need to be transformed before you can use these components in Jest tests.
 
 Add a `transformIgnorePatterns` field with the value `["/node_modules/(?!@ukic/react)"]` to your Jest config.
+
+## Working with slotted SVGs
+
+When wanting to slot an SVG into one of the components, you will need to import SlottedSVG in your React files to prevent an error stating that property 'slot' doesn't exist.
+
+```jsx
+import { SlottedSVG } from "@ukic/react";
+```
+
+<br />
+
+Then update your code to replace any instances of `<svg>` with `<SlottedSVG>`.
+
+As an example, when using IcTopNavigation, if your existing code looks like this:
+
+```jsx
+<IcTopNavigation appTitle="App title">
+    <svg
+        slot="app-icon"
+        ...
+    >
+        <path ... />
+    </svg>
+</IcTopNavigation>
+```
+
+<br />
+
+Then it should be replaced with:
+
+```jsx
+<IcTopNavigation appTitle="App title">
+    <SlottedSVG
+        slot="app-icon"
+        ...
+    >
+        <path ... />
```

Comment:
should the path by passed as a prop on the SlottedSVG eg something like:
<SlottedSVG slot="app-icon" path={path}></SlottedSvg>